lib/misc.js

```diff
@@ -65,3 +65,35 @@ exports.generateId = function(fields) {
   return base32hex.encodeBuffer(hasher.digest(), { paddingChar: '-' });
 };
 
+exports.encryptArrayToFile = function (dataArray, filePath, env, serverSecret) {
+  var iv = `${env}-environment`.substring(0, 16);
+  var key = serverSecret.substring(0, 32);
+  var algorithm = 'aes-256-cbc';
```

Comment:
[nit] Consider pulling these out into a separate function so we guarantee that encryption and decryption use the same values.